atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionManager.java

```diff
@@ -528,7 +530,8 @@ public TransactionService getTransactionService() {
 
     @Override
     public KeyValueServiceStatus getKeyValueServiceStatus() {
-        ClusterAvailabilityStatus clusterAvailabilityStatus = keyValueService.getClusterAvailabilityStatus();
+        ClusterAvailabilityStatus clusterAvailabilityStatus =
+                keyValueService.bestEffortKeyValueService().getClusterAvailabilityStatus();
```

Comment:
I'm leaving this as is, to keep the manager API small for now, but this will need updating.